adapters/handlers/rest/clusterapi/nodes.go

```diff
@@ -32,20 +33,26 @@ func NewNodes(manager nodesManager) *nodes {
 	return &nodes{nodesManager: manager}
 }
 
+var (
+	regxNodes      = regexp.MustCompile(`/status`)
+	regxNodesClass = regexp.MustCompile(`/status/([A-Za-z_-]+)`)
```

Comment:
there is a className regex in the codebase already (`validateClassNameRegex`).
IMO `regxNodesClass` should follow the same rules

Reply:
Agreed, fixed